test.js

```diff
@@ -84,104 +79,99 @@ function test (label, fn) {
       })
     }
 
-    fn.call(ctx, t, function () {
-      var _cleanup = cleanup.bind(ctx, t.end.bind(t))
-      if (ctx.db)
-        return ctx.db.close(_cleanup)
-      _cleanup()
+    openTestDatabase(options, function (err, db) {
```

Comment:
The diff is a bit awkward, but essentially `openTestDatabase` is called first, before a test is executed, instead of all tests calling this function.

Reply:
`?w=1` helps to review :)

Reply:
Hmm, how do you mean?

Reply:
Append that to a GitHub URL and it behaves like `git diff -w`, ignoring whitespace changes.

Reply:
TIL about -w :)